src/tr1d1um/communication.go

```diff
@@ -50,8 +54,24 @@ func (tr1 *Tr1SendAndHandle) Send(ch *ConversionHandler, resp http.ResponseWrite
 	//todo: any more headers to be added here
 	requestToServer.Header.Set("Content-Type", wrp.JSON.ContentType())
 	requestToServer.Header.Set("Authorization", req.Header.Get("Authorization"))
-	respFromServer, err = tr1.timedClient.Do(requestToServer)
```

Comment:
You still want to create a custom `http.Client` with explicit timeouts.  There are two different purposes:

The client's timeouts act as a global protection against server latency.

The context's timeouts define the service boundary.  The context indicates how long your code is willing to wait for an HTTP response at that particular point in code.  Remember that a single context can span mulitple `http.Client` calls.